test/images/pause/pauseloop.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+func main() {
```

Comment:
Tests...?

Also, why do we need this image if we already have a pause image in https://github.com/kubernetes/kubernetes/tree/master/build/pause ? Does that one not work in Windows?

Reply:
Since I'll separate this in a separete PR, will treat that there.

Anyways. That image doesn't have any Windows image in its manifest list, so we can't use that image in testing. Additionally, there's no support for Windows in that image's ``Makefile``, so that's something that will have to be added eventually.

But most likely that the ``pause`` image added here and the one you mentioned do not have quite the same purpose. This image just... pauses (I have been pondering if we can't just use ``agnhost`` for that, since it has a pause functionality - will have to do some digging on it), while the image your proposed is probably meant to be used as the ``--pod-infra-container-image``, which is not quite true for Windows - it must also contain [wincat](https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/cmd/wincat/wincat.go).